### ISSUE TITLE
Google Mobile Ads SDK Causing Crash in iPad App Running on macOS 

### ISSUE THREAD

**Issue by Charchoghlyan** [googleads/googleads-mobile-ios-examples#456]
I am experiencing a crash in my iOS project when using the Google Mobile Ads SDK. The issue occurs specifically when running my iPad app on macOS (Mac Catalyst). Removing the Google Mobile Ads Swift Package Manager (SPM) dependency resolves the issue, but I need to continue using the SDK in my app on iOS/iPadOS.

Failed to look up symbolic reference at 0x1018991b1 - offset 754711 - symbol symbolic _____Sg 14MarketplaceKit14AppDistributorO in /private/var/folders/3p/**/*

**Comment by Xa69** [comment]
I’ll pick this up and give it a shot @Charchoghlyan

**Comment by Charchoghlyan** [comment]
@Xa69 thx, let me know if I can provide more info anything else

**Comment by Xa69** [comment]
@Charchoghlyan from looking the issue up online it looks to be related to the SDK’s dependency on “MarketplaceKit” which is not available for MacOS due to which the app is crashing when run on the Mac Platform.

Reference Link : https://stackoverflow.com/questions/78808081/why-does-running-an-ios-app-through-macos-cause-failed-to-look-up-symbolic-refe

Please try using a downgraded version and see if it works, as per the suggestion in the link I attached downgrading the SDK version worked.

**Comment by Charchoghlyan** [comment]
@Xa69 
11.1.0 is okay
I see only this
Failed to look up symbolic reference at 0x1018991b1 - offset 754711 - symbol symbolic _____Sg 14MarketplaceKit14AppDistributorO in /private/var/folders/3p/**/*

**Comment by Xa69** [comment]
@Charchoghlyan Yes that is an expected error when you try to run it for MacOS as per my above comments, it's because MarketplaceKit is not supported for MacOS only iOS/iPadOS is supported as of now.

**Comment by malandr2** [comment]
Hi @Charchoghlyan, try updating to SDK version [11.9.0](https://developers.google.com/admob/ios/rel-notes), it's expected that should resolve your issue.
